components/footer.tsx

```
@@ -1,5 +1,35 @@
 import Link from "next/link";
-import Image from "next/image"; // Import Image component
+import Image from "next/image";
+import { useTheme } from "next-themes";
+
+import { useTheme } from "next-themes";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTheme'. Consider to delete it or rename it.

'useTheme' is defined here:


(lint/suspicious/noRedeclare)